src/WebJobs.Script.Grpc/Abstractions/DefaultWorkerProcessFactory.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
```

Comment:
This namespace doesn't look right. The default namespace for the project is "Microsoft.Azure.WebJobs.Script.Grpc". Should this namespace then be "Microsoft.Azure.WebJobs.Script.Grpc.Abstractions"?

Reply:
Abstractions is there because I want to expose it a new package with common public interfaces., but have not yet had the time. Things like `JavaWorkerConfig` shouldn't live in the host, they'll be pieces of C# code in the respective language worker nugets (which reference our Abstractions package in order to get the protobuf spec and configuration interfaces).

Reply:
We really need a new project for the common types. @mamaso had that, but to keep things simple for now, ended up merging.

I think the better approach for now would be to rename the project to `WebJobs.Script.Rpc` (or better yet, `WebJobs.Script.LanguageWorkers` or something along those lines) and have the `GRPC` implementation in a folder in that project.